helper-functions/urls.js

```diff
@@ -1,15 +1,15 @@
+const baseURL = 'https://raw.githubusercontent.com/Real-Dev-Squad/website-static/main';
+
 /**
  *
  * @param {string} rdsId
  */
-const getImgURL = (rdsId) =>
-  `https://raw.githubusercontent.com/Real-Dev-Squad/website-static/main/members/${rdsId}/img.png`;
+const getImgURL = (rdsId) => `${baseURL}/members/${rdsId}/img.png`;
 
 /**
  *
  * @param {string} rdsId
  */
-const getDataURL = (rdsId) =>
-  `https://raw.githubusercontent.com/Real-Dev-Squad/website-static/main/members/${rdsId}/data.json`;
+const getDataURL = (rdsId) => `${baseURL}/members/${rdsId}/data.json`;
 
-export { getImgURL, getDataURL };
+export { getImgURL, getDataURL, baseURL };
```

Comment:
Why are we exporting it?

Reply:
Instead of making `https://raw.githubusercontent.com/Real-Dev-Squad/website-static/main/members` this the baseURL I made `https://raw.githubusercontent.com/Real-Dev-Squad/website-static/main` this the base URL so in future we can just add things to this URL like `${baseURL}/ids/mapping.json` etc.

Reply:
Haan, but we should not export it from a utils file. We should put this in constants file and import (can be done in future)

Utils files is only functions, not properties

Reply:
Yes, but we should not export it from a utils file. We should put this in constants file and import (can be done in future)

Utils files is only functions, not properties

Reply:
Make sense, baseURL removed from export.